wormhole/Dockerfile

```diff
@@ -4,19 +4,12 @@ FROM python:3.9.16-slim-bullseye
 # Avoid interactive frontend
 ENV DEBIAN_FRONTEND="noninteractive"
```

Comment:
If you remove the  `apt` runs below, you can also remove this environment variable here :)

Reply:
Indeed. Thank you.